server/data/interfaces/curiousApi/LearnerEducation.ts

```diff
@@ -1,75 +1,181 @@
+   * @description Course start date
+   */
+  learningStartDate?: string
+  /** @description Course Indicator from LRS */
+  level?: string
+  /** @description Number of Guided Learning hours from LRS */
+  lrsGLH?: number
+  /** @description Course Indicator from LRS */
+  miNotionalNVQLevelV2?: string
+  /** @description Course Indicator from LRS */
+  occupationalIndicator?: boolean
+  /** @description Outcome of Course (e.g. Achieved, Partially Achieved etc.) */
+  outcome?: string
+  /** @description Outcome grade of Course (e.g. Passed, Merit, Failed, Distinction etc.) */
+  outcomeGrade?: string
+  /** @description Withdrawal reason if the learner withdraws from course (e.g. Moved to another establishment or release ,ill health etc.) */
+  prisonWithdrawalReason?: string
+  /** @description Course Indicator from LRS */
+  qcfCertificateIndicator?: boolean
+  /** @description Course Indicator from LRS */
+  qcfDiplomaIndicator?: boolean
+  /** @description Course Indicator from LRS */
+  qcfIndicator?: boolean
+  /** @description Indicates if the withdrawal is reviewed */
+  reviewed?: boolean
+  /** @description Course Indicator from LRS */
+  sectorSubjectAreaTier1?: string
+  /** @description Course Indicator from LRS */
+  sectorSubjectAreaTier2?: string
+  /** @description Course Indicator from LRS */
+  subcontractedPartnershipUKPRN?: number
+  /** @description Course Indicator from LRS */
+  unitType?: string
+  /** @description Indicates if withdrawal is agreed or not */
+  withdrawalReasonAgreed?: boolean
+  /** @description Withdrawal reason (defaulted to Other) populated for the courses which are withdrawn */
+  withdrawalReasons?: string
 }
+
+/**
+ * Page
+ * This is the type name as defined in the Curious API, and represents a paged collection of LearnerEducationDTO
+ */
+interface Page {
+  content?: LearnerEducationDTO[]
+  empty?: boolean
+  first?: boolean
+  last?: boolean
+  /** Format: int32 */
+  number?: number
+  /** Format: int32 */
+  numberOfElements?: number
+  pageable?: Pageable
+  /** Format: int32 */
+  size?: number
+  sort?: Sort
+  /** Format: int64 */
+  totalElements?: number
+  /** Format: int32 */
+  totalPages?: number
+}
+
+/**
+ * Sort
+ * This is the type name as defined in the Curious API
+ */
+interface Sort {
+  empty?: boolean
+  sorted?: boolean
+  unsorted?: boolean
+}
+
+/**
+ * Pageable
+ * This is the type name as defined in the Curious API
+ */
+interface Pageable {
+  /** Format: int64 */
+  offset?: number
+  /** Format: int32 */
+  pageNumber?: number
+  /** Format: int32 */
+  pageSize?: number
+  paged?: boolean
+  sort?: Sort
+  unpaged?: boolean
+}
+
+/**
+ * `LearnerEductionPagedResponse` is an alias for the Curious API type `Page`
+ * `LearnerEductionPagedResponse` is a better name as it more accurately describes the fact that the data is part
+ * of a paged response of Learner Education data from the Curious API.
+ */
+export type LearnerEductionPagedResponse = Page
```

Comment:
Here I am exporting 2 types - `LearnerEducationPagedResponse` which is the entire API response; and `LearnerEducation` which is a single course or qualification

Reply:
Much better like this 👍